utils/useEtherscanUrl.ts

```diff
@@ -12,12 +12,18 @@ const NETWORK_MAP: NetworkMap = {
   mainnet: "https://etherscan.io",
 };
 
-export const useEtherscanUrl = (txnHash: string | null) => {
+export const useEtherscanUrl = (hex: string | null) => {
   const { network } = Connection.useContainer();
-  if (!txnHash || !network) {
+
+  if (!network || !hex || !network.name) {
     return undefined;
   }
-
   const baseUrl = NETWORK_MAP[network.name] || NETWORK_MAP["mainnet"];
-  return `${baseUrl}/tx/${txnHash}`;
+  if (hex.length == 66) {
```

Comment:
66 and 42 are a bit unreadable to me, can you perhaps define them as constants ADDRESS_HEX_LENGTH and TX_HASH_HEX_LENGTH ?

Reply:
how about I simply add a comment? having these as extra variables is pretty redundant imho.

Reply:
I just think using variables in functions rather than hard coded values is preferable. I also imagine we will likely use these byte lengths in other future components

Reply:
Just throwing in my opinion here, but I think it's fine to have the hardcoded magic numbers since it is pretty clear what it's doing and it's in a small isolated module. It's also something that will never change (i.e. addresses are always going to be 42 in length).